kanalony-web-server/app/model/Implicits.scala

```diff
@@ -0,0 +1,13 @@
+package model
+
+import argonaut.Argonaut._
+
+/**
+ * Created by elad.benedict on 2/21/2016.
+ */
+object Implicits {
```

Comment:
It exposes the implicit evidence required by the Argonaut library for JSON serialization and deserialization (decodeOption and asJson methods)